### PR TITLE
Add RC adjustments for nav_fw_cruise_thr and nav_fw_pitch2thr

### PR DESCRIPTION
**Pull request by shellixyz**
Answers #2719 request and also adds RC adjustment for nav_fw_pitch2thr

### CODE REVIEW

**Comment by shellixyz** [comment]
Adjustment step of cruise throttle should probably be 10 or 20 instead of 1.

File: src/main/fc/rc_adjustments.c [hunk 1]
```diff
@@ -156,6 +158,14 @@ static const adjustmentConfig_t defaultAdjustmentConfigs[ADJUSTMENT_FUNCTION_COU
         .adjustmentFunction = ADJUSTMENT_MANUAL_YAW_RATE,
         .mode = ADJUSTMENT_MODE_STEP,
         .data = { .stepConfig = { .step = 1 }}
+    }, {
+        .adjustmentFunction = ADJUSTMENT_NAV_FW_CRUISE_THR,
+        .mode = ADJUSTMENT_MODE_STEP,
+        .data = { .stepConfig = { .step = 1 }}
```
Comment (digitalentity) [inline src/main/fc/rc_adjustments.c]: Yes, this should probably be 10

**Review by digitalentity — APPROVED** [review]

**Comment by digitalentity** [comment]
@shellixyz is there Configurator support for this PR?

**Comment by shellixyz** [comment]
I didn't do it yet because the configurator support for the new manual settings would need to be merged first but there is not much to do. It's almost ready only cosmetic issues remain. I will make a pull-request soon to support all the new settings.

**Comment by shellixyz** [comment]
The configurator update is ready: https://github.com/iNavFlight/inav-configurator/pull/338

I also added the Yaw expo adjustment and the manual pitch & roll rate adjustment for consistency.

**Pull request merged**